src/elements-experimental/seat-reservation/seat-reservation-area/seat-reservation-area.scss

```diff
@@ -0,0 +1,46 @@
+@use '@sbb-esta/lyne-elements/core/styles' as sbb;
+
+:host {
+  --sbb-reservation-area-width: 1;
+  --sbb-reservation-area-height: 1;
+  --sbb-reservation-area-rotation: 0;
+  --sbb-reservation-area-border-radius: #{sbb.px-to-rem-build(4)};
+
+  display: inline-block;
+  position: inherit;
+}
+
+.sbb-sr-area {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: calc(var(--sbb-reservation-area-width) * 1px);
+  height: calc(var(--sbb-reservation-area-height) * 1px);
+  rotate: calc(var(--sbb-reservation-area-rotation) * 1deg);
+  border: #{sbb.px-to-rem-build(1)} solid var(--sbb-color-graphite);
+  background-color: var(--sbb-color-white);
+  border-radius: var(--sbb-reservation-area-border-radius);
+  box-sizing: border-box;
+
+  :host([background='DARK']) & {
+    background-color: var(--sbb-color-milk);
+  }
+
+  :host([mounting='UPPER_BORDER']) & {
+    border-block-start-width: #{sbb.px-to-rem-build(2)};
```

Comment:
the border with of 2 should probably be a CSS variable as it is duplicated. OR just use lyne design token `var(--sbb-border-width-2x)`